impl.go

```diff
@@ -287,16 +295,25 @@ func validReceiver(recv string) bool {
 }
 
 func main() {
-	if len(os.Args) != 3 {
+	flag.Parse()
+
+	if len(flag.Args()) < 2 {
 		fmt.Fprint(os.Stderr, usage)
 		os.Exit(2)
 	}
-	recv, iface := os.Args[1], os.Args[2]
+
+	recv, iface := flag.Arg(0), flag.Arg(1)
 	if !validReceiver(recv) {
 		fatal(fmt.Sprintf("invalid receiver: %q", recv))
 	}
 
-	fns, err := funcs(iface)
+	if *flagSrcDir == "" {
+		if dir, err := os.Getwd(); err != nil {
```

Comment:
Are you planning on taking this?

It's been a few days and it's still not fixed, and I'm just trying to figure out whether I need to send a PR or not.